activerecord/test/schema/schema.rb

```diff
@@ -992,16 +992,19 @@
   create_table :records, force: true do |t|
   end
 
-  if supports_foreign_keys?
-    # fk_test_has_fk should be before fk_test_has_pk
-    create_table :fk_test_has_fk, force: true do |t|
-      t.integer :fk_id, null: false
-    end
+  if supports_foreign_keys_in_create?
```

Comment:
If `create_table` is called with foreign keys, generates in single query. ref #20009.